src/App.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Footer } from "./index";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+function App() {
+  return (
+    <>
+      <header
+        style={{
+          display: "flex",
+          justifyContent: "space-between",
+          alignItems: "center",
+          padding: "1rem",
+          borderBottom: "solid 1px rgba(0,0,0,0.2)",
+        }}
+      >
+        <h1 style={{ fontSize: "25px", margin: 0 }}>Council Data Project</h1>
+        <section>
+          <a href="#" style={{ padding: "1rem" }}>
+            Events
+          </a>
+          <a href="#" style={{ padding: "1rem" }}>
+            People
+          </a>
+        </section>
+      </header>
+      <main
+        style={{
+          display: "flex",
+          flexFlow: "column wrap",
+          justifyContent: "center",
+          alignItems: "center",
+          height: "85vh",
+        }}
+      >
+        <h1 style={{ fontSize: "30px", marginBottom: "2rem" }}>Search City Council Transcripts</h1>
+        <section>
+          <input type="text" style={{ marginRight: "1rem", width: "30vw" }}></input>
+          <button
+            style={{
+              border: "2px solid transparent",
+              color: "#ffffff",
+              fontSize: "1rem",
+              fontWeight: "bold",
+              background: "#0060df",
+              padding: "8px 24px",
+              borderRadius: "4px",
+            }}
+          >
+            Search
+          </button>
+        </section>
+      </main>
+      <footer>
+        <Footer
+          footerLinksSections={[
+            {
+              footerLinksSectionName: "Seattle",
+              links: [
+                { label: "City Of Seattle", url: "#" },
+                { label: "Seattle Open Data", url: "#" },
+                { label: "Seattle Channel", url: "#" },
+              ],
+            },
```

Comment:
Nit: just make these "Lorem", "Ipsum" and the section be "Links"

Reply:
As a future thought for the Footer actually, I think we should allow the link sections to be optional because right now it's required to pass something in.

Reply:
Yea that makes sense to me. Didn't realize that.